File: src/BukkitPE/event/weather/WeatherEvent.php
<?php
namespace BukkitPE\event\weather;

use BukkitPE\event\Event;
use BukkitPE\level\Level;

abstract class WeatherEvent extends Event{
    private $level;

    public function __construct(Level $level){
        $this->level = $level;
    }

    public function getLevel(){
        return $this->level;
    }
}

File: src/main/java/net/BukkitPE/event/level/WeatherEvent.java
package net.BukkitPE.event.level;

import net.BukkitPE.event.Event;
import net.BukkitPE.level.Level;

/**
 * author: funcraft
 * BukkitPE Project
 */
public abstract class WeatherEvent extends Event {

    private Level level;

    public WeatherEvent(Level level) {
        this.level = level;
    }

    public Level getLevel() {
        return level;
    }
}
